src/app/groups/containers/add-group/add-group.component.scss

```diff
@@ -0,0 +1,5 @@
+@import 'src/assets/scss/functions';
+
+.title {
+  margin: toRem(38) 0 toRem(16) 0;
```

Comment:
It looks strange to me that the style of a "classical" title has to be defined here while it looks like it is the same everywhere (looks like it is re-used), but you know better than me the situation.